app/assets/javascripts/index/contextmenu.js

```diff
@@ -74,6 +74,66 @@ OSM.initializeContextMenu = function (map) {
     }
   });
 
+  const expiry = new Date();
+  expiry.setYear(expiry.getFullYear() + 10);
+  let selectedDarkModeFilter = Cookies.get("_osm_dark_mode_filter") || "brightness80";
+
+  let darkModeMenuElements = $(map.contextmenu.addItem({
+    separator: true
+  }));
+
+  const darkModeMenuItems = [
+    {
+      name: "brightness100",
+      filter: "none"
+    },
+    {
+      name: "brightness80",
+      filter: "brightness(.8)"
+    },
+    {
+      name: "brightness60",
+      filter: "brightness(.6)"
+    },
+    {
+      name: "invert",
+      filter: "invert(.8) hue-rotate(180deg)"
```

Comment:
I think .8 has pretty weak contrast and would suggest .95 instead. Or provide several options, like .8, .9, .95 and 1.